soc/riscv/nordic_nrf/nrf54h/CMakeLists.txt

```diff
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Nordic Semiconductor
+# SPDX-License-Identifier: Apache-2.0
+
+# Ensure that image size aligns with 16 bytes so that MRAMC finalizes all writes
+# for the image correctly
+zephyr_linker_sources(SECTIONS SORT_KEY zzz_place_align_at_end align.ld)
```

Comment:
This is probably relevant for Arm cores as well.

Reply:
@anangl ^